src/table/styles.scss

```diff
@@ -113,9 +113,18 @@
   /* used in test-utils */
 }
 
+/*
+The min/max/width token values in in Visual Refresh should align 
+the table header and body cells selection control with the table
+filter search icon.
+*/
 .selection-control {
-  position: relative;
   box-sizing: border-box;
+  max-width: awsui.$size-table-selection-horizontal;
```

Comment:
Does it make sense to define `max-width` and `min-width` to the same value that is already used as `width`?

Reply:
I'm not sure, to be honest. This is the way the original code was written so I just stuck with it lest I create unintended consequences.

Reply:
Right, I see